viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -298,8 +298,8 @@ export class LighthouseReportViewer {
       placeholder.remove();
     }
 
-    if (window.ga) {
-      window.ga('send', 'event', 'report', 'view');
+    if (window.gtag) {
+      window.gtag('event', 'report', {type: 'view'});
```

Comment:
looking at [this](https://developers.google.com/analytics/devguides/collection/ga4/events?client_type=gtag&sjid=15586495081079579696-NC#google_tag_gtagjs_overview) i think the ones in this should be like

`window.gtag?.('event', 'view')`

basically getting rid of 'report'